src/main/resources/reference.conf

```diff
@@ -676,11 +688,26 @@ resourceTypes = {
       ingest_data = {
         description = "Add data to a dataset"
       }
+      soft_delete = {
+        description = "Soft delete dataset data"
+      }
+      hard_delete = {
+        description = "Hard delete dataset data"
+      }
       update_data = {
-        description = "Add or delete data for a dataset"
+        description = "(deprecated) Add or delete data for a dataset"
       }
       create_datasnapshot = {
-        description = "Create a datasnapshot that includes this dataset"
+        description = "(deprecated) Create a datasnapshot that includes this dataset"
+      }
+      link_snapshot = {
+        description = "Create a snapshot that uses this dataset"
+      }
+      unlink_snapshot = {
+        description = "Delete a snapshot that uses this dataset"
+      }
+      list_snapshots = {
```

Comment:
Are any of the roles below supposed to have this action?

Reply:
Ah yeah, doh.  I missed that.  nice catch!